.travis.yml

```diff
@@ -1,11 +1,20 @@
 language: node_js
 node_js:
-  - 0.10
+- 0.1
```

Comment:
Not sure this is correct, maybe rather change to a recent version of node? Like 6 or 8